src/test/run-make-fulldeps/coverage-reports/expected_show_coverage.uses_crate.txt

```diff
@@ -19,12 +19,12 @@
    18|      2|    println!("used_only_from_bin_crate_generic_function with {:?}", arg);
    19|      2|}
   ------------------
-  | used_crate::used_only_from_bin_crate_generic_function::<&str>:
```

Comment:
This is not unusual for Windows to swap these, and should be ignored by the diff flags.